.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Can you reuse eslint-config-algolia? https://github.com/algolia/eslint-config-algolia